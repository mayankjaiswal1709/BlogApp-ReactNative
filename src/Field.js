import React from 'react';
import {TextInput} from 'react-native';


const Field = (props) => {
    return (
        <TextInput {...props} 
        style={{borderRadius:70 ,paddingHorizontal:35, paddingVertical:20,width:'70%' , height:60,backgroundColor:'#000',marginVertical:10,color:"white"}} 
        placeholderTextColor='white'
        >
            
        </TextInput>
    );
}

export default Field;
