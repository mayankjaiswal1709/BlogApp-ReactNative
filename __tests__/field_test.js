import 'react-native'
import React from 'react'
import Field from '../src/Field'

import renderer from 'react-test-renderer'

test('test render snapshot', ()=>{
    const snapshot =renderer.create(<Field/>).toJSON();
    expect(snapshot).toMatchSnapshot();
});