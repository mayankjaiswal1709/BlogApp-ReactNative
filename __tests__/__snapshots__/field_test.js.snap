// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render snapshot 1`] = `
<TextInput
  placeholderTextColor="white"
  style={
    {
      "backgroundColor": "#000",
      "borderRadius": 70,
      "color": "white",
      "height": 60,
      "marginVertical": 10,
      "paddingHorizontal": 35,
      "paddingVertical": 20,
      "width": "70%",
    }
  }
/>
`;
